safe-global/safe-wallet-web
# Feat: allow custom recovery delay

## What it solves

Resolves #3489

## How this PR fixes it
- Add an option in the recovery period dropdown for a custom delay.
- Open a number input on selection

## How to test it
- Go through the steps for setting up account recovery
- Make sure the next button is disabled when no recovery delay, or an invalid recovery delay, is provided.
- Make sure the custom delay shows up correctly in the transaction review step and in account settings after executing the tx.
- Make sure the correct delay is used when executing recovery from the second account.

## Screenshots
<img width="702" alt="image" src="https://github.com/safe-global/safe-wallet-web/assets/17801424/fd48da73-8d21-4848-b732-80fb2d382b10">

## Checklist
